### PR TITLE
Create Db and make queries

### PR DESCRIPTION
**Pull request by niicode**
In this milestone, I was able to achieve the following:

- Create the vet_clinic database
- Create an animals table
- Populate the table with data
- Make database queries
[ScreenShot One](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.03.51%20AM.png)
[ScreenShot two](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.11.23%20AM.png)
[ScreenShot three](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.12.03%20AM.png)
[ScreenShot four](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.12.39%20AM.png)
[ScreenShot five](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.13.13%20AM.png)
[ScreenShot six](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.13.40%20AM.png)
[ScreenShot Seven](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.14.10%20AM.png)
[ScreenShot Eight](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.16.03%20AM.png)
[ScreenShot Nine](https://github.com/niicode/vet_clinic/blob/db-setup/screenshots/Screenshot%202022-10-18%20at%208.25.36%20AM.png)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT NOT NULL,
```
Comment (Nemwel-Boniface) [inline schema.sql:4]: - [ ] [OPTIONAL] You could optionally make the ID as auto-incremented and the primary key.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @niicode     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Good job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] As requested in the project requirements, you are required to add the screenshots of your query results in the pull request description and not in the screenshots folder. Kindly add them to your pull request description.

image


_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (niicode) [reply schema.sql:4]: Hi, @Nemwel-Boniface Thanks for the detailed review, I already have the queries in my queries.sql file I am not sure what you meant by I am supposed to copy my queries again. Can you please explain further, Thank you

Comment (Nemwel-Boniface) [reply schema.sql:4]: Hello @niicode I did not quite get you here. But I can see that you included the query results screenshots in your pull request description which are perfectly fine.

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @niicode     :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes I requested good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**